### PR TITLE
feat: add new matomo event

### PR DESCRIPTION
**Pull request by solidovic**
### Description

Max button in withdrawals request form

Снимок экрана 2023-10-06 в 17 17 00

### Checklist:

- [x] Checked the changes locally.
- [x] Created / updated analytics events.
- [ ] Created / updated the technical documentation (README.md / [docs](https://docs.lido.fi/) / etc.).
- [ ] Affects / requires changes in other services (Matomo / Sentry / CloudFlare / etc.).

### CODE REVIEW

File: shared/forms/components/input-decorator-max-button/input-decorator-max-button.tsx [hunk 1]
```diff
@@ -1,19 +1,42 @@
...
 
 export const InputDecoratorMaxButton = ({
   disabled,
   onClick,
 }: InputDecoratorMaxButtonProps) => {
+  const thisComponentInsideWithdrawalsRequestFormRef = useRef(false);
+
+  try {
+    void useRequestFormData();
+    thisComponentInsideWithdrawalsRequestFormRef.current = true;
```
Comment (Jeday) [inline shared/forms/components/input-decorator-max-button/input-decorator-max-button.tsx:24]: We can track this event in InputComponent much easier. Sure we might need additional onMax event handler. But that would be so much better than this.

**Review by Jeday — CHANGES_REQUESTED** [review]

File: shared/forms/components/input-decorator-max-button/input-decorator-max-button.tsx (hunk 1 above)
Comment (solidovic) [reply shared/forms/components/input-decorator-max-button/input-decorator-max-button.tsx:24]: decided: use props drilling

Comment (solidovic) [reply shared/forms/components/input-decorator-max-button/input-decorator-max-button.tsx]: done

File: config/matomoClickEvents.ts [hunk 2]
```diff
@@ -37,8 +37,6 @@ export const enum MATOMO_CLICK_EVENTS_TYPES {
   wrapTokenSelectETH = 'wrapTokenSelectEth',
   // Unwrap tab
   l2BannerUnwrap = 'l2BannerUnwrap',
-  // /rewards page
-  calculateRewards = 'calculateRewards',
```
Comment (DiRaiks) [inline config/matomoClickEvents.ts:41]: Why are you deleting this?

**Review by DiRaiks — APPROVED** [review]

**Review by DiRaiks — APPROVED** [review]

**Review by itaven — APPROVED** [review]

**Pull request closed without merging**